src/layout/container.rs

```diff
@@ -199,7 +203,7 @@ impl Container {
     /// Determines if the container is focused or not
     pub fn is_focused(&self) -> bool {
         match *self {
-            Container::Root { .. } => false,
```

Comment:
Ah that was something that slipped by and probably didn't fix for consistency. 

Regardless, I'd rather keep the Root to only be a tuple so as to mark that it shouldn't really be extended.